### PR TITLE
Don't set indexing vars when metrics-endpoint is set

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description

When using metrics-endpoint we shouldn't set any environment variables used for indexing

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: cmd/ocp.go [hunk 1]
```diff
@@ -80,17 +80,22 @@ func openShiftCmd() *cobra.Command {
...
+			"GC":         fmt.Sprintf("%v", gc),
+			"GC_METRICS": fmt.Sprintf("%v", gcMetrics),
 		}
+		envVars["LOCAL_INDEXING"] = fmt.Sprintf("%v", *localIndexing)
 		if alerting {
 			envVars["ALERTS"] = "alerts.yml"
+		} else {
+			envVars["ALERTS"] = ""
+		}
+		// If metricsEndpoint is not set, use values from flags
```
Comment (vishnuchalla) [inline cmd/ocp.go:95]: Are we making sure that we are giving higher precedence to `metricEndpoint` over ES CLI args in the kube-burner core repo too?

Comment (rsevilla87) [reply cmd/ocp.go:95]: There're not ES CLI args in kube-burner core, they only exist in `kube-burner-ocp` (for now)

Comment (vishnuchalla) [reply cmd/ocp.go:95]: Looks like there is one: https://github.com/kube-burner/kube-burner/blob/main/cmd/kube-burner/kube-burner.go#L356. Which can be removed I guess. wdyt?

Comment (vishnuchalla) [reply cmd/ocp.go:95]: Yes it doesn't. But when we have metric endpoint do we want to keep es-server as a CLI arg?

Comment (rsevilla87) [reply cmd/ocp.go:95]: Not sure if Im following you... as I said before, the ES_SERVER and ES_INDEX env vars are not used by kube-burner core. I've added a small hint in the `--metrics-endpoint` help to note this behavior

**Review by vishnuchalla — APPROVED** [review]
As discussed offline, approving these changes.

**Pull request merged**